Bundle/SeoBundle/Controller/PageSeoController.php

```diff
@@ -72,8 +71,12 @@ public function settingsAction(View $page)
             $this->congrat('victoire_seo.save.success');
 
             //redirect to the page url
-            $pageUrl = $page->getUrl();
-            $url = $this->generateUrl('victoire_core_page_show', array('url' => $pageUrl));
+            if($page instanceof BusinessTemplate)
```

Comment:
BusinessTemplates are not the only Views to do not have an url, please treat the case of Template too